examples/run_setup_tutorial.py

```diff
@@ -0,0 +1,100 @@
+"""
+=========
+Run Setup
+=========
+
+By: Jan N. van Rijn
+
+One of the key features of the openml-python library is that is allows to
+reinstantiate flows with hyperparameter settings that were uploaded before.
+This tutorial uses the concept of setups. Although setups are not extensively
+described in the OpenML documentation (because most users will not directly
+use them)
+
+A key requirement for reinstantiating a flow is to have the same scikit-learn
+version as the flow that was uploaded. This tutorial will upload the flow
+itself, so it can be ran with any scikit-learn version that is supported by
+this library.
+
+In this tutotial we will
+    1) Create a flow and use it to solve a task
+    2) Download the flow, reinstantiate the model with same hyperparameters,
+       and solve the same task again.
+    3) We will verify that the obtained results are exactly the same.
+Readers interested in reinstantiating a setup can skip part 1 and 2 and start
+with part 3 immediately.
```

Comment:
Doesn't part 3 only compare the two arrays?

Reply:
Yes, but is is quite an important point and it combines pt 1 and pt 2. I would prefer to keep it in a separate part, albeit only one line of code (it could be extended with more checks later, if someone is motivated)

Reply:
You are right. I removed the sentence.